straxen/plugins/veto_pulse_processing.py

```diff
@@ -21,6 +21,11 @@
         default=20, track=True,
         help='Minimum hit amplitude in ADC counts above baseline. '
              'Specify as a tuple of length n_nveto_pmts, or a number.'),
+    strax.Option(
+        'min_samples_alt_baseline_nv',
+        default=500, track=True,
```

Comment:
I do see how this is going to create funny artifacts given the low levelness of this data. Also, a 1us long pulse is not really that long is it?

Perhaps you can also plot in https://xe1t-wiki.lngs.infn.it/doku.php?id=xenon:xenonnt:wenz:comissioning:nveto:baseline:nveto_baseline what the effect on short pulses would be?

Reply:
See my comments in the wiki.